ISSUE_LABELS.md

```diff
@@ -0,0 +1,120 @@
+# W3DT issue labeling standards
+
+The W3DT org standardizes on the following default core labels for GitHub issues and pull requests. This ensures continuity
+between repos, helping to improve the overall experience for contributors as a whole while also making it easier to use project
+planning tools (such as ZenHub) to create landscape-level views across multiple repos.
+
+## Mandatory labels
+
+### Global
+These are exceptions to the remainder of this labeling taxonomy, but exist for continuity with global GitHub practices for new contributors.
+
+| Label              | Description                                                  | Color     |
+| -----              | -----------                                                  | -----     |
+| `bounty`           | Has bounty! See https://github.com/ipfs/devgrants/projects/1 | `#1cfc60` |
+| `good first issue` | Good issue for new contributors                              | `#7057ff` |
+| `help wanted`      | Seeking public contribution on this issue                    | `#0e8a16` |
+
+### Priority
+Indicates priority as a function of standard PL-wide OKR priority rankings. **Important: P0 items need an assignee to act as a DRI.**
+
+| Label | Description                                               | Color     |
+| ----- | -----------                                               | -----     |
+| `P0`  | Critical: Tackled by core team ASAP                       | `#b60205` |
+| `P1`  | High: Likely tackled by core team if no one steps up      | `#d93f0b` |
+| `P2`  | Medium: Good to have, but can wait until someone steps up | `#e99695` |
+| `P3`  | Low: Not priority right now                               | `#f9d0c4` |
+
+### Kind
+Overarching type of issue or PR. For an additional layer of specificity, use the `area` label.
+
+| Label               | Description                                                             | Color     |
+| -----               | -----------                                                             | -----     |
+| `kind/architecture` | Core architecture of project                                            | `#c7def8` |
+| `kind/bug`          | A bug in existing code (including security flaws)                       | `#fc2929` |
+| `kind/discussion`   | Topical discussion; usually not changes to codebase                     | `#c7def8` |
+| `kind/enhancement`  | A net-new feature or improvement to an existing feature                 | `#c7def8` |
+| `kind/maintenance`  | Work required to avoid breaking changes or harm to project's status quo | `#c7def8` |
```

Comment:
Were we going to add a note to be more specific here?  Maybe something about "Build systems, dependency upgrades, CI."

Reply:
Good point. The current description better describes "tech debt" which probably deserves a different label.

Reply:
It looks like we need to split this into:

- `kind/maintenance`: CI, labeling, permissions.
- `kind/cleanup` (or `kind/debt`): Cleanup, refactors, etc.

Reply:
Sounds good to me!